test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -393,6 +412,23 @@ proc ::AlgorandGoal::AccountTransfer { FROM_WALLET_NAME FROM_WALLET_PASSWORD FRO
     return $TRANSACTION_ID
 }
 
+# Account Transfer, rekey
+proc ::AlgorandGoal::AccountTransferRekey { FROM_WALLET_NAME FROM_WALLET_PASSWORD FROM_ACCOUNT_ADDRESS TRANSFER_AMOUNT TO_ACCOUNT_ADDRESS FEE_AMOUNT REKEY_ACCOUNT_ADDRESS TEST_PRIMARY_NODE_DIR } {
```

Comment:
Note that this isn't optimal as it's the same as the `AccountTransfer` procedure plus an additional argument.  It's good enough for now, but this should be revisited after the release.